server/hotrod/src/test/resources/create_certs.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+echo "==== Removing old files ===="
+find . -name "*.jks" -exec rm -rf {} \;
+
+echo "==== Generating private keys ===="
+keytool -genkey -noprompt -trustcacerts -keyalg RSA -alias "default" -dname "CN=HotRod_1, OU=Infinispan, O=JBoss, L=Red Hat, ST=World, C=WW" -keypass "secret" -storepass "secret" -keypass "secret2" -keystore "password_server_keystore.jks"
```

Comment:
well, I have no strong opinion, but I'd incline to @tristantarrant proposal. Tests are sometimes source of knowledge how to configure some functionality and for security setup, which is quite often source of confusion, complex/descriptive tests can have this added value that in case of questions/confusion we can easily point users to the test suite to explain how to use/setup given feature.